.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
+
```

Comment:
As this will be the first point of contact for some users, I wonder if this is a little abrupt. We just want to give pointers to the available information, really. How about something like "Tskit is a free and open-source project that welcomes contributions from everyone. Here are some resources to help you get started:

- Developer documentation
- [etc]

We ask all users to follow our [code of conduct]() when interacting with the project.
"

Reply:
I've also added a note about the slack group in https://github.com/tskit-dev/tskit/pull/915/commits/bdbd6e14bb9e3518d83ae7a3ad418545409b40ef